app/bundles/FormBundle/Model/FormModel.php

```diff
@@ -437,8 +437,8 @@ public function generateHtml(Form $entity, $persist = true): string
         $formToRender  = '@MauticForm/Builder/form.html.twig';
 
         if (!empty($theme)) {
-            if ($this->twig->getLoader()->exists('@themes/'.$theme.'/html/MauticFormBundle/Builder/_style.html.twig')) {
-                $styleToRender = '@themes/'.$theme.'/html/MauticFormBundle/Builder/_style.html.twig';
+            if ($this->twig->getLoader()->exists('@themes/'.$theme.'/html/MauticFormBundle/Builder/style.html.twig')) {
+                $styleToRender = '@themes/'.$theme.'/html/MauticFormBundle/Builder/style.html.twig';
```

Comment:
I'm not clear about the reason, but it should be with the underscore as that's how the file is called now: 

https://github.com/mautic/mautic/blob/5.x/app/bundles/FormBundle/Resources/views/Builder/_style.html.twig

Reply:
Hm.. It's an unannounced breaking change.


 I'd rather opt for renaming the core file.

I mean, why change this, after all? :man_shrugging:  Especially, because it seems inconsistent to me: Some files have underscores, some not.

Reply:
I'm OK with renaming it back. I tracked it down to https://github.com/mautic/mautic/pull/11898/files#diff-10644a08b8c71352a404ceba8363170cdc45c7411eeabd89bfff43891e05ab06L13 but still don't see a reason for it. Feel free to rename it back.

Reply:
Some time ago, I found [this](https://symfony.com/doc/current/templates.html#including-templates) mentioning the use of underscores in twig/symfony:

> the `_` prefix is optional, but it's a convention used to better differentiate between full templates and template fragments

Is there anyone who makes the call about naming conventions?

Reply:
Clearly, there isn't :D For PHP we have PHP CS Fixer rules, but Twig has not Code Style set as of now.